shared/actions/login/saga.js

```diff
@@ -275,8 +275,8 @@ const displayAndPromptSecretSaga = onBackSaga =>
       yield Saga.call(onBackSaga)
       return EngineRpc.rpcCancel(InputCancelError)
     } else if (qrScanned || textEntered) {
-      const phrase = qrScanned ? qrScanned.payload.phrase : textEntered.payload.phrase
-      return EngineRpc.rpcResult({phrase, secret: null})
+      const phrase: HiddenString = qrScanned ? qrScanned.payload.phrase : textEntered.payload.phrase
```

Comment:
racers are untyped here :(

Reply:
turns out qrScanned was a hiddenstring, and textEntered was a plain string , which wont' work